admin/javascript/src/silverstripe-backend.js

```diff
@@ -45,8 +45,15 @@ class SilverStripeBackend {
    * @return object - Promise
    */
   post(url, data) {
-    return this.fetch(url, { method: 'post', credentials: 'same-origin', body: data })
-      .then(checkStatus);
+    return this.fetch(url, {
+      method: 'post',
+      headers: new Headers({
```

Comment:
@flashbackzoo from the looks of things, Headers is added to the globals when `isomorphic-fetch` is included. :-/ https://github.com/matthew-andrews/isomorphic-fetch/blob/master/fetch-npm-node.js